web/src/main/webapp/WEB-INF/classes/setup/sql/data/data-db-default.sql

```diff
@@ -678,3 +679,11 @@ INSERT INTO HarvesterSettings (id, parentid, name, value) VALUES  (1,NULL,'harve
 INSERT INTO Users (id, username, password, name, surname, profile, kind, organisation, security, authtype) VALUES  (1,'admin','46e44386069f7cf0d4f2a420b9a2383a612f316e2024b0fe84052b0b96c479a23e8a0be8b90fb8c2','admin','admin',0,'','','','');
 INSERT INTO Address (id, address, city, country, state, zip) VALUES  (1, '', '', '', '', '');
 INSERT INTO UserAddress (userid, addressid) VALUES  (1, 1);
+
+
+-- ======================================================================
+-- === Table: MetadataURNTemplates
+-- ======================================================================
+
+INSERT INTO MetadataIdentifierTemplate (id, name, template, isdefault) VALUES  (0, 'Custom URN', '', 'y');
+INSERT INTO MetadataIdentifierTemplate (id, name, template, isdefault) VALUES  (1, 'Autogenerated URN', '', 'y');
```

Comment:
Does this mean that by default users will have 2 choices ? Would be good to keep things simple by default. ie. having only the autogenerated UUID like we have now. 

The 2 options above are both defaults ?